PlugIns/DotScene/README.md

```diff
@@ -42,4 +43,36 @@ To add logic properties to the scene you can use the `<userData>` node as follow
         <property data="1.0" name="mass_radius" type="float" />
     </userData>
 </entity>
-```
\ No newline at end of file
+```
+
+## How to use DotScene
+In recent OGRE3D versions DotScene has been incorporated into the main OGRE repo and made into a Plugin.
+So it has to be loaded as another OGRE Plugin.
+
+In `plugins.cfg`, add the following line:
+```
+Plugin=Plugin_DotScene
+```
+
+Include the header `#include <OgreDotSceneLoader.h>`, located in: `OgreSDK\ogre-1.12.11\include\OGRE\Plugins\DotScene`
```

Comment:
the header is not needed either

Reply:
I tried removing the header but then I complains about incomplete type:
```
C:\OGRE\Built\ogre-1.12.11\include\OGRE\OgreConfigFile.h|94|note: declared here|
C:\OGRE\Dev\ShaderTest\ShaderTest.cpp|188|error: incomplete type 'Ogre::Codec' used in nested name specifier|
```

Reply:
`#include <OgreCodec.h>`

Reply:
or even better: `#include <Ogre.h>`

Reply:
Isn't it overkill to `#include <Ogre.h>`?

I thought it was better to include only what you need.

Reply:
> I thought it was better to include only what you need.

generally, I would say no. There might be special cases, like headers that get included 1000x times (literally) which you want to keep small or compile-time increases.
But as a user of Ogre, you likely want for stuff to just work which Ogre.h gives you.

Also, it gives me the freedom to drop classes without breaking your code. E.g. OgreVector3.h is only kept for backward compatibility.

Reply:
OK